kbcstorage/configurations.py

```diff
@@ -0,0 +1,114 @@
+"""
+Manages calls to the Storage API relating to configurations
+
+Full documentation https://keboola.docs.apiary.io/#reference/components-and-configurations
+"""
+from kbcstorage.base import Endpoint
+
+
+class Configurations(Endpoint):
+    """
+    Configurations Endpoint
+    """
+
+    def __init__(self, root_url, token, branch_id):
+        """
+        Create a Component endpoint.
+
+        Args:
+            root_url (:obj:`str`): The base url for the API.
+            token (:obj:`str`): A storage API key.
+            branch_id (str): The ID of branch to use, use 'default' to work without branch (in main).
+        """
+        super().__init__(root_url, f"branch/{branch_id}/components", token)
+
+    def detail(self, component_id, configuration_id):
+        """
+        Retrieves information about a given configuration.
+
+        Args:
+            component_id (str): The id of the component.
+            configuration_id (str): The id of the configuration.
+
+        Returns:
+            response_body: The parsed json from the HTTP response.
+
+        Raises:
+            requests.HTTPError: If the API request fails.
+        """
+        if not isinstance(component_id, str) or component_id == '':
+            raise ValueError("Invalid component_id '{}'.".format(component_id))
+        if not isinstance(configuration_id, str) or configuration_id == '':
+            raise ValueError("Invalid component_id '{}'.".format(configuration_id))
+        url = '{}/{}/configs/{}'.format(self.base_url, component_id, configuration_id)
```

Comment:
Do we want to accept empty strings?  Doesn't the detail call require values?

Reply:
If component_id is an empty string, it raises an error, doesn't it?

Reply:
omg, sorry, I don't know how to read a simple conditional statement 🤦